projects/configs/tag-operations/pet.yaml

```diff
@@ -0,0 +1,222 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+- name: Series Date
+  group: 0x0008
+  element: 0x0021
+  op: replace
+- name: Acquisition Date
+  group: 0x0008
+  element: 0x0022
+  op: replace
+- name: Series Time
+  group: 0x0008
+  element: 0x0031
+  op: keep
+- name: Acquisition Time
+  group: 0x0008
+  element: 0x0032
+  op: keep
```

Comment:
Should we be replacing the times by default? Dave [suggested](https://github.com/SAFEHR-data/PIXL/pull/592#issue-2779926957) we default to keeping them:

> More information about the acquisition (reconstrution, randoms correction, etc) is also useful to ensure study protocol compliance in single and multi-centre studies. Acquisition time is a Level 1 attribute for PET series. And I think it may be worth keeping time to ensure uptake time, especially as in the PET base module all of the dates are either replace or wiped out.

Reply:
I think we're probably okay but might be worth dropping Sophie an email to get her take on it from an IG perspective

Reply:
Yeah good idea. Sophie confirmed it's okay to keep the times